__std__/cells/plutus/library/make-plutus-project.nix

```diff
@@ -0,0 +1,201 @@
+{ inputs, cell }:
+{ compiler-nix-name ? inputs.cells.toolchain.library.ghc-compiler-nix-name }:
```

Comment:
the default `? inputs.cells.toolchain.library.ghc-compiler-nix-name` is never triggered now, do we want to remove that file?

Reply:
Yeah, I guess it's not that useful indeed.

Reply:
It's still used in a few places. I'll leave it for now.